docs/docs/concepts/retrieval_unstructured.mdx

```diff
@@ -0,0 +1,160 @@
+# Retrieval for Unstructured Data
+
+## Motivation
+
+Language models (LLMs) are trained on vast but fixed datasets, which limits their ability to access up-to-date or domain-specific information. 
+
+To enhance their performance on specific tasks, we can augment their knowledge using retrieval systems.
+
+Retrieval systems fetch relevant information from external sources, which can then be included in the prompt given to the model. 
+
+Key benefits of using retrieval systems include:
+
+1. Access to recent or private information
+2. Improved accuracy on domain-specific tasks
+3. Reduced hallucination by grounding responses in retrieved facts
+4. Cost-effective alternative to fine-tuning for factual recall
+
+## Un-Structured Data
+
+Vector stores are a type of retrieval system that work with unstructured data. 
+
+Unstructured documents (e.g., raw text, tables, images) are often embedded into a compressed representation (e.g., a vector) that can used directly for search based upon similarity to an input query.
+
+## Query Translation  
```

Comment:
Query translation -> Query analysis?

translation could mean: 
* translate from english to french
* translate from raw language to SQL

----

In case there's any relevant information here: https://python.langchain.com/v0.1/docs/use_cases/query_analysis/